modules/publicize/ui.php

```diff
@@ -114,12 +115,55 @@ public static function connected_notice( $service_name ) { ?>
 		</div><?php
 	}
 
+	public static function add_disconnect_notice() {
+		add_action( 'admin_notices', array( __CLASS__, 'disconnected_notice' ) );
+	}
+
+	public static function disconnected_notice() {
+		echo "<div class='updated'>\n";
+		echo '<p>' . esc_html(__( 'That connection has been removed.', 'jetpack' ) ) . "</p>\n";
+		echo "</div>\n\n";
+	}
+
 	public static function denied_notice() { ?>
 		<div class='updated'>
 			<p><?php _e ( "You have chosen not to connect your blog. Please click 'accept' when prompted if you wish to connect your accounts.", 'jetpack' ); ?></p>
 		</div><?php
 	}
 
+	function admin_page_warning() {
```

Comment:
Does `admin_page_warning()` need to go into ui.php? That file is shared between WP.com and Jetpack.